### ISSUE TITLE
Use tabulator's own ajax loader to load data at some places

### ISSUE THREAD

**Issue by answerquest** [WRI-Cities/static-GTFS-manager#24]
See `/xml2GTFS.html` Stations section. Tabulator is loading the data via ajax GET request by itself. The advantage : less coding needed in `/js/xml2GTFS.js` 

It also has on-load functions to trigger things after the data has loaded or if it fails to load. See http://tabulator.info/docs/3.4#callbacks-ajax 

We can do this on all the pages where a tabulator table has to be populated on page load.

**Comment by answerquest** [comment]
implemented this for nearly all tables by v2.0.0.

**Comment by answerquest** [comment]
Oops, spoke too soon. it's been done for stops and routes tables, but not implemented yet at some easy places like calendar table. Have to do some work on this.

**Comment by answerquest** [comment]
Tables covered so far where this applies: stops, routes, agency, calendar, translations, fare_attributes
to check :fare rules pivoted and simple.

**Comment by answerquest** [comment]
Done; fare rules simple can do it, but not fare rules pivoted as that one's construction itself takes place only AFTER the data is loaded in over ajax.
Will be updated with v2.1.0

**Comment by answerquest** [comment]
oh heck its done with the v2.0.2 release itself.
